cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -482,7 +482,7 @@ WorkflowStatistics getWorkflowStatistics(Id.Workflow workflowId, long startTime,
    * @param timeInterval The timeInterval with which the user wants to space out the runs
    * @return Map of runId of Workflow to DetailedStatistics of the run
    */
-  Collection<WorkflowDataset.WorkflowRunRecord> retrieveSpacedRecords(Id.Workflow workflow, String runId,
+  Collection<WorkflowDataset.WorkflowRunRecord> retrieveSpacedRecords(Id.Program workflow, String runId,
```

Comment:
Only `Id.Program` will be used from now on, rather than both `Id.Program` and `Id.Workflow`, etc. `Id.Workflow` will no longer be instantiable and will only contain helper methods.